staging/src/k8s.io/pod-security-admission/admission/admission.go

```diff
@@ -677,6 +683,29 @@ func (a *Admission) exemptRuntimeClass(runtimeClass *string) bool {
 	// TODO: consider optimizing to O(1) lookup
 	return containsString(*runtimeClass, a.Configuration.Exemptions.RuntimeClasses)
 }
+
+// Filter and prioritise pods based on runtimeclass and uniqueness of the controller respectively for evaluation
+func (a *Admission) prioritisePods(pods []*corev1.Pod) []*corev1.Pod {
+	var replicatedPods []*corev1.Pod
+	var prioritisedPods []*corev1.Pod
```

Comment:
can slightly optimize the memory by making the return list use the same allocated memory (document that the pods list passed in is modified):

`prioritizedPods := pods[:0]`